### DISCUSSION TITLE
Does anyone have frame rate issues with Chrome?

### DISCUSSION THREAD

**Discussion by AiOO** [letmaik/pyvirtualcam#86] [Q&A]
Hello! I'm making [QX100-Webcamizer](https://github.com/AiOO/QX100-Webcamizer) which is webcam-related project based on this project.

First of all, thanks for making a nice and insightful project.

Is there anyone like me who has a problem with a significantly lower frame rate in Chrome? When I tested it, it works normally in other browsers. I'm looking for someone who's experiencing the same problem, or someone who's solved this problem.

I've confirmed that the frame rate is 20 fps when using OBS virtual camera in a normal way.

Belows are my environments:

- Model: Macbook Air (M1, 2020)
- CPU: Apple M1
- RAM: 16GB
- OBS 27.2.4 (Intel)
- Chrome 103.0.5060.53
- Firefox 102.0

And this is the capture of my problem (left is Chrome, another is Firefox):

https://user-images.githubusercontent.com/1644576/177008029-526f11c0-515b-47d5-8817-e2789cfdf176.mp4

**Comment by AiOO** [comment]
Update: it's a bothersome way, but this issue can be bypassed with [Snap Camera](https://snapcamera.snapchat.com/) or [XSplit VCam](https://www.xsplit.com/vcam).

https://user-images.githubusercontent.com/1644576/177024184-ac2af373-f05a-4fa9-bb7d-b6c3d3ee5c57.mov

**Comment by letmaik** [comment]
I tried it on Edge and Chrome and couldn't reproduce the issue with the `animated_gradient.py` example. I'm on Windows though, so it's not really comparable. Maybe someone else can confirm the issue.

**Reply by AiOO** [reply]
Thank you for your confirmation. I'll share something new when I find out.
